contracts/DelFiPrice.sol

```diff
@@ -105,6 +152,23 @@ contract DelFiPrice is OpenOracleView {
         }
     }
 
+
+    /**
+     * @notice Returns the cToken address for symbol
+     * @param symbol The symbol to map to cToken address
+     * @return cToken The cToken address for the given symbol
+     */
+    function getOracleKey(string memory symbol) public view returns (address cToken) {
+        if (keccak256(abi.encodePacked(symbol)) == keccak256(abi.encodePacked("ETH"))) return cEthAddress;
```

Comment:
we could keccack the arg symbol once at the beginning, and put the symbols for comparison in contants to make this more palatable

Reply:
eg constant symbolDai = keccak256(abi.encodePacked("DAI")) etc etc